### PR TITLE
Fix wraparound if progress runs for more than 24hrs

### PR DESCRIPTION
**Pull request by gakera**
The duration string will now display the number of days the task has been running.

### CODE REVIEW

File: src/ShellProgressBar/target.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
+(function ($R) {
```
Comment (Mpdreamz) [inline src/ShellProgressBar/target.js]: Thanks for the PR! Can you remove this file? 

Curious how did this file make it in here?

**Comment by gakera** [comment]
I was doing some unrelated experiment. Does it prevent you from merging the
other stuff?

**Comment by Mpdreamz** [comment]
It does yeah, happy to merge this in but that file needs to be gone 😄

**Comment by gakera** [comment]
Done

**Comment by 0xced** [comment]
Wouldn't it be simpler to just use the `ToString()` method of `TimeSpan`?

**Comment by Mpdreamz** [comment]
Merged and released: https://www.nuget.org/packages/ShellProgressBar/4.3.0

@0xced it would be but even simpler was merging and releasing which I've opted for now,will revisit at some point

**Pull request merged**